build/vsts_build.yaml

```diff
@@ -561,6 +561,15 @@ phases:
       testRunTitle: "NuGet.Client End To End Tests "
     condition: "succeededOrFailed()"
 
+  - task: PowerShell@1
+    displayName: "Kill running instances of DevEnv"
+    inputs:
+      scriptType: "inlineScript"
+      inlineScript: |
+        . $(Build.Repository.LocalPath)\\scripts\\e2etests\\VSUtils.ps1
+        KillRunningInstancesOfVS
```

Comment:
Hope build won't fail if it failed to kill either of the process?

Reply:
nope